src/components/shared/EthAmount.tsx

```diff
@@ -0,0 +1,47 @@
+import { Tooltip } from 'antd'
+
+import { BigNumber } from 'ethers'
+import { formatWad, parseWad } from 'utils/formatNumber'
+
+import CurrencySymbol from './CurrencySymbol'
+
+import { CURRENCY_ETH } from 'constants/currency'
+
+import ETHToUSD from './ETHToUSD'
+
+// Eth amount which displays the equiv USD amount in a tooltip on hover
+export default function ETHAmount({
+  amount,
+  precision,
+  padEnd,
+}: {
+  amount?: BigNumber | string
+  precision?: number
+  padEnd?: boolean
+}) {
+  // If 0 < amount < 1, set precision to 2
+  const betweenZeroAndOne = (amount: BigNumber) => {
```

Comment:
@johnnyd-eth perhaps we can move this  "pure" function outside of the component scope?

Reply:
Yep sounds good